modules/pencil.js/readme.md

```diff
@@ -51,18 +51,16 @@ const scene = new Pencil.scene();
 // Just the part you need
 import Scene from "@pencil.js/scene";
 // or
-import { Scene } from "pencil.js";
+import Scene from "pencil.js";
```

Comment:
I'm not sure if this was done by me or from pulling upstream I can revert the import changes if need be

Reply:
This is an older version of the code from upstream. Trying merging first.